client/src/pages/Homepage/Homepage.js

```diff
@@ -4,30 +4,27 @@ import './Homepage.css';
 class Homepage extends Component {
   render() {
     return (
-      <div id="content" className="container-fluid">
+      <div id="content">
         <div className="row">
           <div className="col-md-5 align-self-center" id="nswnIs">
             <h1>NSWN is </h1>
             <p id="underscore"></p>
           </div>
         </div>
-    </div>
-      // <div className="App">
-      //   <header className="App-header">
-      //     <p>
-      //       Edit <code>src/App.js</code> and save to reload.
-      //     </p>
-      //     <a
-      //       className="App-link"
-      //       href="https://reactjs.org"
-      //       target="_blank"
-      //       rel="noopener noreferrer"
-      //     >
-      //       Learn React
-      //     </a>
-      //   </header>
-      //   <p className="App-intro">{this.state.apiResponse}</p>
-      // </div>
+
+        <div class="container-fluid parallax">
+          <div className="text-center" id="imageText">
+            <h1>I am John Doe</h1>
+            <p>And I'm a Photographer</p>
+          </div>
+        </div>
```

Comment:
Can we create a separate parallax component and then import that? Figured we might want to add it on other pages.